### ISSUE TITLE
Examining Model Fit Statistics

### ISSUE THREAD

**Issue by paganantonio** [jmbh/mgm#17]
In your thorough and helpful article published in 2019, you indicate various functions such as resample and predict. Is there a function that can be used to examine the model fit of the tvmgm model? I know you use alpha-level to conclude the significance of the model fit, but wondered if a method could be conducted to examine model fit by using BIC or AIC type metrics.
Thanks!

**Comment by jmbh** [comment]
We estimate the time-varying mgm with nodewise regression, so we don't have a joint likelihood that would be required to compute a BIC/AIC. However, you could add the likelihoods of all models together to obtain a pseudo log likelihood.

Alternatively, you can use out of sample / out of bag prediction errors, pooled over nodes, to perform model selection. This is what is implemented in the function bwSelect().

Also note that we use no p-values in the mgm package. The only alpha-parameter used in mgm() is the alpha determining the mix between L1 and L2 penalties in the elastic net penalty.

**Comment by paganantonio** [comment]
Thank you for the response and clarification! With respect to the bwSelect() function, what is constituted as "out of sample/out of bag." Are two separate data sets required for this analysis? 
Thanks again!

**Comment by jmbh** [comment]
Apologies for the late reply. The bwSelect() function splits the provided dataset into a train and test set. To be able to assess different bandwidth parameters for a time-varying model spanning the entire time series, this split is performed across the whole period of the time series (as opposed to, for example, taking the first 80% of the time series as the training set).
